CONTRIBUTING.md

```diff
@@ -92,15 +195,125 @@ than waiting for a full **NCL Entry** to be completed.
+comprehensive guides. They are intended to provide more extensive testing than is useful
+in an NCL entry. Receipts are intended to be the minimal amount of documentation necessary to
+add an entry to the NCL Index in cases where a full **NCL Entry** is not
+necessary or where providing an initial entry to the NCL Index is more important
+than waiting for a full **NCL Entry** to be completed.
 
 1. Create a new file in `ncl/ncl_receipts/` based off of the
    `templates/receipt_template.ipynb` template.
 
-1. See below for adding the covered functions to the NCL Index
+1. Create raw `.ncl` script within `ncl/ncl_raw/` to store `NCL Code` section
+
+1. Remove placeholder and descriptive text below `NCL Code`,
+`Python Functionality`, `Comparison`
 
 1. Make sure to clear and run all outputs before asking for a review
 
-### Adding to the NCL Index
+1. Add a new line to the `ncl/ncl_index/ncl-index-table.csv` file
+
+[Example of a NCL Receipt](https://ncar.github.io/geocat-applications/ncl/receipts/days_in_month.html)
+
+### Generate the Documentation Locally
+At the base of the reposistory `geocat-applications/`, run
+```
+make clean html
+```
+This will build the documentation locally so you can view your changes.
 
-Add a new line to the `ncl/ncl_index/ncl-index-table.csv` file
+New documentation can be viewed by opening `index.html` generated under
+ `_build/html/index.html` on a local browser:
+
+```
+# on Mac
+open _build/html/index.html
+
+# Otherwise, open with a specific browser, like Firefox
+firefox _build/html/index.html
+```
+Running this commands on the terminal will open a new browser tab with the
+changes you have made to the repository. The webpage runs on your local host,
+but all the pages are linked together and act as they will on the
+`geocat-applications` website. You can browse your changes, navigate with links,
+and view images as you would on the official website.
+
+## Contribute the Code
+Once you have completed your changes on notebooks and they are ready for review
+by the GeoCAT team, you can open a new Pull Request. This section describes how
+to open a pull request and what to expect after you open it.
+
+### Check Files Changed
+
+For a Python Entry, the files changed should include:
+- New Python page under relevant `applications` folder
+- Link to notebook in `.rst` file in the same directory as `applications/` (for
+example: if new notebook is part of `data_analysis`, add link to
+`applications/data_analysis/data_analysis.rst`)
+- Link new notebook in `applications/applications.rst` file
+
+For a NCL to Python entry, the files changes should include:
+- New NCL to Python notebook under `ncl/ncl_entries`
+- Link to new `ncl_entries` notebook in `ncl/ncl_index/ncl_entries.rst`
+- A new row in `ncl/ncl_index/ncl-index-table.csv` for each NCL function
+- A new receipts entry under `ncl/receipts/`
+- A new raw `.ncl` script to store `NCL Code` for receipt within `ncl/ncl_raw/`
+
+### Open a New Pull Request
```

Comment:
It looks like this might be missing the push changes section and there's a lot of learning content in here as well. Is this also a section we could lean on Project Pythia Foundations for?  Maybe we could do something like [GeoCAT-comp](https://geocat-comp.readthedocs.io/en/stable/contrib.html#contribute-your-code) with some additional links to Project Pythia Foundations for extra support.